README.md

```diff
@@ -38,16 +34,42 @@ In this project we leverage and modify two external libraries:
 which is available only for server-side applications. This version is based on a fork from [Scality](https://github.com/scality/kubernetes-client-javascript/tree/browser) which added browser-side support.
 * The laboratory operator leverages the [Kubevirt](https://kubevirt.io/) library to create the VirtualMachineInstances.
 
-## Install
+# Installation
+
+## Preliminary Note
 
 CrownLabs can be installed on any Kubernetes cluster, although with a non-negligible degree of adaptation.
 This would require a non-trivial knowledge of how Kubernetes (and the wonderful world of microservices) works.
 No magic install procedure is unfortunately available (yet).
 
 In a nutshell, you have to install all the components with your own custom configuration files, which may largely depend upon your physical install.
-A huge degree of customization is possible in this respect: pure data-link vs. BGP-based load balancing, the number (and capabilities) of your servers, the desired degree of high availability, integration with external authentication servers, creation of admin/user credentials, your own secrets to protect the internal communication among the components.
+A huge degree of customization is possible in this respect: pure data-link vs. BGP-based load balancing, the number (and capabilities) of your servers, the desired degree of high availability, integration with external authentication servers, creation of admin/user credentials, your own secrets to protect the internal communication among the components. 
 
 Do not expect to complete this task in a few hours; likely, you may need several days, or even more.
 Help is available on our Slack channels.
-
 For more information, visit the CrownLabs website: [https://crownlabs.polito.it](https://crownlabs.polito.it).
+
+# Pre-Requirements
+
+Crownlabs have been specifically designed for bare-metal clusters and this assumption will be adopted across the documentation. To deploy CrownLabs, we have to rely on a full-fledged Kubernetes cluster where at least a subset of node supports Hardware Virtualization. 
```

Comment:
"CrownLabs has been ..."
"... a subset of nodes supports ..."

Reply:
eheheh